resources/serverless/charts/webhook/templates/deployment.yaml

```diff
@@ -99,10 +104,8 @@ spec:
               {{ include "tplValue" ( dict "value" .Values.container.envs.configObservabilityName "context" . ) | nindent 14 }}
             - name: KUBERNETES_MIN_VERSION # TODO: Delete this env after all CI pipelines are using k8s >1.17.0
               value: "1.16.0"
-            - name: LOG_LEVEL
-              value: {{ .Values.container.envs.logLevel.value }}
-            - name: LOG_FORMAT
-              value: {{ .Values.container.envs.logFormat.value }}
+            - name: APP_FILE_CONFIG_PATH
```

Comment:
Shouldn't we put it in `configmap.yaml`?
I created PR with removing obsolete envs: https://github.com/kyma-project/kyma/pull/15786

Reply:
I don't understant. What do you want to put in `configmap.yaml`? Right now we move the `logLevel` and the `logFormat` into the `config.yaml` file and put the path to this file under the `APP_FILE_CONFIG_PATH` env.

Reply:
We set envs in webhook using configmap. `APP_FILE_CONFIG_PATH` is env